public/js/chat.js

```diff
@@ -17,6 +11,8 @@ var version = VERSION;
 var blop = new Audio('sounds/blop.wav');
 var regex = /(&zwj;|&nbsp;)/g;
 
+var user, timer, socket, oldname, username, typeTimer;
```

Comment:
Hi Steve, our code style is to place each new variable declaration into a new line, with the `var` keyword in front of each variable. Please fix this, thanks!